packages/commercetools-sdk-middleware-auth/src/build-requests.js

```diff
@@ -12,34 +12,34 @@ type BuiltRequestParams = {
   body: string;
 }
 
-const defaultAuthHost = 'https://auth.sphere.io'
-
 // POST https://{host}/oauth/token?grant_type=client_credentials&scope={scope}
 // Authorization: Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==
 export function buildRequestForClientCredentialsFlow (
-  {
-    host = defaultAuthHost,
-    projectKey,
-    credentials,
-    scopes,
-  }: AuthMiddlewareOptions = {},
+  options: AuthMiddlewareOptions,
 ): BuiltRequestParams {
-  if (!projectKey)
+  // TODO: use a better validator
```

Comment:
How were you able to a _TODO_ comment though, I thought our eslint config forbids that...🤔

Nevermind, I just saw the _:_ that makes the rule useless 🙄